libs/parallelism/algorithms/tests/regressions/ranges_facilities.cpp

```diff
@@ -0,0 +1,41 @@
+//  Copyright (c) 2021 Giannis Gonidelis
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <hpx/hpx_main.hpp>
+#include <hpx/modules/testing.hpp>
+#include <hpx/parallel/util/ranges_facilities.hpp>
+
+#include "libs/parallelism/algorithms/tests/util/iter_sent.hpp"
```

Comment:
@hkaiser  Please also check this moved header.

Reply:
Should we move that to <hpx/parallel/util/iter_sent_test_facilities.hpp>? @msimberg any suggestions?

Reply:
I'd like to keep it out of the install if possible. @gonidelis is this file common only to the `libs/parallelism/algorithms` tests? If yes, I'd suggest we do a simple `#include "../../util/iter_sent.hpp"`. Alternatively we add `target_include_directories(${test_name} PRIVATE ../../include)` for all the tests in `libs/parallelism/algorithms` and put it under unique directory in `libs/parallelism/tests/include`.

Reply:
@gonidelis this is the last thing that should be fixed before this PR can be merged. Could you have a look, please?

Reply:
@msimberg @hkaiser There are only two tests outside of algorithms that uses this and they are the `libs/core/iterator_support/tests/unit/is_sentinel_for.cpp`/`is_sized_sentinel.hpp` ones. So do you suggest I go with the cmake option?

Reply:
In that case please move the file to the `iterator_support` tests directory (really an appropriate `include/...` subdirectory), and use `target_include_directories` on the tests that need the directory.

Reply:
@msimberg over